src/Ookii.Dialogs.Wpf/Ookii.Dialogs.Wpf.csproj

```diff
@@ -1,51 +1,68 @@
+    <Authors>Sven Groot, Caio Proiete, AceOfAces</Authors>
+    <Summary>Common dialog classes for WPF applications.</Summary>
+    <Description>Ookii.Dialogs.Wpf is a class library for .NET and .NET Core applications providing several common dialogs. Included are classes for task dialogs, credential dialogs, progress dialogs, and common file dialogs.</Description>
+    <PackageProjectUrl>https://github.com/augustoproiete/ookii-dialogs-wpf</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/acemod13/ookii-dialogs-wpf</RepositoryUrl>
+    <RepositoryType></RepositoryType>
+    <PackageTags>ookii dialogs wpf windows-presentation-foundation progress-dialog task-dialog credential-dialog common-file-dialog</PackageTags>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageIconUrl></PackageIconUrl>
+    <NeutralLanguage>en</NeutralLanguage>
+    <AssemblyVersion>2.1.20307.0</AssemblyVersion>
+    <FileVersion>2.1.20307.0</FileVersion>
+    <Copyright>Copyright (c) Sven Groot (Ookii.org) 2009.</Copyright>
+    <Company>Ookii.org</Company>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <TieredCompilationQuickJit>true</TieredCompilationQuickJit>
+    <TieredCompilation>true</TieredCompilation>
+    <PackageIcon>ookii-dialogs-wpf-nuget.png</PackageIcon>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
-  <PropertyGroup>
-    <SignAssembly>true</SignAssembly>
-  </PropertyGroup>
-  <PropertyGroup>
-    <AssemblyOriginatorKeyFile>..\..\assets\ookii-dialogs.snk</AssemblyOriginatorKeyFile>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <ShouldCreateLogs>True</ShouldCreateLogs>
+    <AdvancedSettingsExpanded>False</AdvancedSettingsExpanded>
+    <UpdateAssemblyVersion>True</UpdateAssemblyVersion>
+    <UpdateAssemblyFileVersion>True</UpdateAssemblyFileVersion>
+    <UpdateAssemblyInfoVersion>False</UpdateAssemblyInfoVersion>
+    <AssemblyVersionSettings>None.None.DateStamp.None</AssemblyVersionSettings>
+    <UpdatePackageVersion>True</UpdatePackageVersion>
+    <AssemblyInfoVersionType>SettingsVersion</AssemblyInfoVersionType>
+    <InheritWinAppVersionFrom>None</InheritWinAppVersionFrom>
+    <AssemblyFileVersionSettings>None.None.DateStamp.None</AssemblyFileVersionSettings>
+    <PackageVersionSettings>AssemblyVersion.None.None</PackageVersionSettings>
   </PropertyGroup>
+
   <ItemGroup>
+    <None Remove="CredentialDialog.bmp" />
+    <None Remove="CustomDictionary.xml" />
+    <None Remove="ProgressDialog.bmp" />
+    <None Remove="TaskDialog.bmp" />
+    <None Remove="VistaFolderBrowserDialog.bmp" />
+    <None Remove="VistaOpenFileDialog.bmp" />
+    <None Remove="VistaSaveFileDialog.bmp" />
+    <None Include="..\..\assets\ookii-dialogs-wpf-nuget.png">
+      <Pack>True</Pack>
+      <PackagePath></PackagePath>
+    </None>
+    <None Include="..\..\LICENSE">
```

Comment:
This relates to my other question on NuGet CLI w/ `.nuspec` vs. `dotnet pack` w/ `.csproj` - if we are going with the latter, then let's use [PackageLicenseExpression](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagelicenseexpression) with a valid [SPDX license identifier](https://spdx.org/licenses/) here instead of embedding a custom license file